solr/core/src/java/org/apache/solr/handler/component/MoreLikeThisComponent.java

```diff
@@ -127,6 +128,53 @@ public void process(ResponseBuilder rb) throws IOException {
     }
   }
 
+  private static NamedList<String> putMLTIntoParamList(
```

Comment:
We strive to match the Google Java Style guide, and that means using "Mlt" and not "MLT" inside names.  But maybe MLT is so pervasive capitalized this way in the code already (before striving for this) that you are trying to be consistent?  Any way; do as you wish.

Reply:
We have a hundred of occurrences of `MLT`, it might be great idea, but out of scope.

Reply:
ok. I put it as `mltViaQueryParams`